python/llm/src/ipex_llm/transformers/npu_models/mp_models_base.py

```diff
@@ -383,7 +383,7 @@ def update_cache(self, past_key_value, indexes):
             self.load_cache_async()
 
     def load_cache_async(self):
-        self.load_wt_fn(len(self.input_ops), self._mm, self.kv_cache_c_handle)
+        self.load_wt_fn(len(self.input_ops), self._mm, self.kv_cache_c_handle, verify_size=True)
```

Comment:
Not sure how much runtime overhead of `verify_size`, if it is large, maybe we can disable it at runtime and only use it when debugging?

Reply:
> Not sure how much runtime overhead of `verify_size`, if it is large, maybe we can disable it at runtime and only use it when debugging?

According to my experiment, the overhead seems quite small. BTW have removed it : )